src/test/java/com/google/api/codegen/testdata/discoveries/go/go_adexchangebuyer.v1.4.json.baseline

```diff
@@ -49,7 +49,7 @@ func getClient(ctx context.Context) (*http.Client, error) {
   // https://foo.com/bar
   //
   // Authorize using the following scope:
-  //    adexchangebuyer.AdexchangeBuyerScope
+  //     adexchangebuyer.AdexchangeBuyerScope
```

Comment:
Here the scope is a Go identifier, whereas in Java it is an http URI. I assume the value of the Go identifier is the corresponding URI.

But this makes me wonder whether for consistency we should quote the URIs in the comments in the other languages.

Thoughts?